worblehat-domain/src/main/java/de/codecentric/psd/worblehat/domain/Book.java

```diff
@@ -107,6 +116,18 @@ public Borrowing getBorrowing() {
 		return borrowing;
 	}
 
+	public String getBookCoverImageURL() {
+		return bookCoverImageURL;
+	}
+
+	public void setBookCoverImageURL(String bookCoverImageURL) {
+		this.bookCoverImageURL = bookCoverImageURL;
+	}
+
+	public boolean hasBookCover() {
+		return !(this.bookCoverImageURL == null) && !this.bookCoverImageURL.isEmpty();

```

Comment:
return this.bookCoverImageURL != null && !this.bookCoverImageURL.isEmpty(); does the same